sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/baseSender.ts

```diff
@@ -155,6 +155,11 @@ export abstract class BaseSender {
       } else if (restError.statusCode && isRetriable(restError.statusCode)) {
         this.networkStatsbeatMetrics?.countRetry(restError.statusCode);
         return this.persist(envelopes);
+      } else if (restError.statusCode === 400 && restError.message.includes("Invalid instrumentation key")) {
+        const invalidInstrumentationKeyError = new Error("Invalid instrumentation key");
+        this.incrementStatsbeatFailure();
+        diag.error(invalidInstrumentationKeyError.message);
```

Comment:
This code is not needed, if you don't add lines 161 and 162, line 179 will log the error